azure-pipelines.yml

```diff
@@ -0,0 +1,108 @@
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+
+schedules:
+  - cron: "0 0 */14 * *"
+    displayName: "Bi-monthly build"
+    branches:
+      include:
+        - master
+        - benchmarking
+
+
+
+trigger:
+- master
+- benchmarking
+
+jobs:
+
+- job: CPU_Benchmarks
+
+  pool:
+    name: Default
+    demands: 
+      - GPUs -equals 0
+      - Agent.OS -equals Linux
+  steps:
+
+  - script: |
+      conda env remove --yes -n tomopy-bench
+      rm -rf tomopy.github.io
+    displayName: Remove existing build environment
+
+  - bash: echo "##vso[task.prependpath]$CONDA/bin"
+    displayName: Add conda to PATH
```

Comment:
It's weird that we are adding conda to the path after already using conda once. Swap these two steps?